scm-ui/ui-types/src/RepositoryRole.ts

```diff
@@ -22,17 +22,19 @@
  * SOFTWARE.
  */
 
-import { Links, PagedCollection } from "./hal";
+import {HalRepresentation, PagedCollection} from "./hal";
 
-export type RepositoryRole = {
+export type RepositoryRoleBase = {
   name: string;
   verbs: string[];
   type?: string;
   creationDate?: string;
   lastModified?: string;
```

Comment:
`creationDate` and `lastModified` should not be part of the base type, because they should not be used for a creation.